Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM node:14-slim
+FROM node:14-slim as build
 
 LABEL maintainer="Orta Therox"
 LABEL "com.github.actions.name"="Danger JS Action"
 LABEL "com.github.actions.description"="Runs JavaScript/TypeScript Dangerfiles"
 LABEL "com.github.actions.icon"="zap"
 LABEL "com.github.actions.color"="blue"
 
-RUN mkdir -p /usr/src/danger
-COPY . /usr/src/danger
-RUN cd /usr/src/danger && \
-  yarn && \
-  yarn run build:fast && \
-  chmod +x distribution/commands/danger.js && \
-  ln -s $(pwd)/distribution/commands/danger.js /usr/bin/danger
+WORKDIR /usr/src/danger
+RUN yarn global add yarn-audit-fix
```

Comment:
This should really be a separate process.

like using dependabot.

Beacuase now, you have something at a gitref (the pkg lock file) but it results in different outcome in the resulting build.